src/alfasim_score/converter/alfacase/score_input_reader.py

```diff
@@ -97,3 +140,96 @@ def read_lithology_materials(self) -> List[Dict[str, Union[Scalar, str]]]:
                 }
             )
         return lithology_data
+
+    def read_packer_fluid(self) -> List[Dict[str, Union[Scalar, str]]]:
+        """ "Get the properties of fluid above packer."""
+        # TODO PWPA-1970: review this fluid default with fluid actually used by SCORE file
+        # the fluid used now is water
+        return [
+            {
+                "name": FLUID_DEFAULT_NAME,
+                "type": "fluid",
+                "density": Scalar(1000.0, "kg/m3", "density"),
+                "thermal_conductivity": Scalar(0.6, THERMAL_CONDUCTIVITY_UNIT),
+                "specific_heat": Scalar(4181.0, SPECIFIC_HEAT_UNIT),
+                "thermal_expansion": Scalar(0.0004, THERMAL_EXPANSION_UNIT),
+            }
+        ]
+
+    def read_casings(self) -> List[Dict[str, Any]]:
+        """Read the data for the casing from SCORE input file"""
+        casing_data = []
+        for item in self.input_content["operation"]["thermal_simulation"]["well_strings"]:
+            if item["interval"] != WellItemFunction.OPEN.value:
+                casing_data.append(
+                    {
+                        "type": WellItemType(item["type"]) if "type" in item else WellItemType.NONE,
+                        "function": WellItemFunction(item["interval"]),
+                        "hanger_md": Scalar(item["hanger_md"], LENGTH_UNIT, "length"),
+                        "shoe_md": Scalar(item["shoe_md"], LENGTH_UNIT, "length"),
+                        "final_md": Scalar(item["final_md"], LENGTH_UNIT, "length"),
+                        "top_of_cement": Scalar(item["toc_md"], LENGTH_UNIT, "length"),
+                        "hole_diameter": Scalar(item["hole_size"], DIAMETER_UNIT, "diameter"),
+                        "sections": [
+                            {
+                                "material": section["pipe"]["grade"]["name"],
+                                "top_md": Scalar(section["top_md"], LENGTH_UNIT, "length"),
+                                "base_md": Scalar(section["base_md"], LENGTH_UNIT, "length"),
+                                "inner_diameter": Scalar(
+                                    section["pipe"]["od"] - 2.0 * section["pipe"]["wt"],
+                                    DIAMETER_UNIT,
+                                    "diameter",
+                                ),
+                                "outer_diameter": Scalar(
+                                    section["pipe"]["od"], DIAMETER_UNIT, "diameter"
+                                ),
+                            }
+                            for section in item["string_sections"]
+                        ],
+                    }
+                )
+        return casing_data
+
+    def read_tubing(self) -> List[Dict[str, Union[Scalar, str]]]:
+        """Read the data for the tubing from SCORE input file"""
```

Comment:
OK. It was supposed to be performed in PWPA-1931.
Thus, include this task in commit.

Reply:
Indeed, I forgot about the other task hahahaha good point.